docs/release-notes.md

```diff
@@ -1,230 +1,124 @@
+<p class="divider"></p>
+## Note From the Team
 
-v2.0.0 - 2022-03-02: 
+Hey everyone,
 
-- **tls-configuration:** add support for TLS Connection type (#0bcddb6) 
-- **BREAKING CHANGE:** messageID is no longer passed in to functions and is tracked internally
+It has been a very quick 6 weeks and the team is very excited to deliver Open AMT Cloud Toolkit version 2.3.  Unlike previous releases, every component of the Toolkit has been updated in some way.  If you haven't had a chance yet, I encourage you to watch the release video where Mike provides some highlights from this release.  Additionally, we've added a [Video Walkthroughs](https://open-amt-cloud-toolkit.github.io/docs/2.3/videos/) section to our documentation where Bryan Wendlandt has been creating video tutorials of our [Getting Started Guide](https://open-amt-cloud-toolkit.github.io/docs/2.3/GetStarted/prerequisites/).  Watch that space for additional video content to be delivered there.  Don't forget to like and subscribe!
 
-v1.7.0 - 2022-02-24
+Find out [what's new](#whats-new) and delve into the [details](#get-the-details) below-- and enjoy our new release of the toolkit.
 
-- **general-settings:** adds PUT method for AMT_GeneralSettings (#f39c0df) 
+*Best wishes,*<br>
+*Matt Primrose | Product Owner | The Open AMT Cloud Toolkit Team*
+<br>
+<p class="divider"></p>
 
-v1.6.0 - 2022-02-23
+## What's New?
 
-- **amt:** adds WiFiPortConfigurationService to amt. (#d2a2f0e) 
+:material-star:** Customer Request: RPC-Go as a library**
 
-v1.5.0 - 2022-02-15
+We have added an option to build RPC as a library, allowing developers to import RPC into their applications as a dll or shared object.  Instead of deploying and executing RPC as an application, developers can call and monitor RPC directly within their own application.
 
-- **amt:** adds delete method to ManagementPresenceRemoteSAP and PublicKeyCertificate (#919794e) 
+:material-auto-fix:** Improvement: RPC-Go Refactoring**
 
-v1.4.1 - 2022-02-08
+Our Go version of Remote Provision Client has undergone significant refactoring to eliminate all C/C++ code.  By transitioning this to 100% Go code we remove the requirement of the GCC toolchain to build RPC, you can cross-compile RPC on Linux or Windows and use it on any Go supported OS (example: we tested on Fedora and it worked without any code changes), and the build time is greatly reduced.  
 
-- **exports:** align exports with expected types from breaking change in PR[#29](https://github.com/open-amt-cloud-toolkit/wsman-messages/issues/29) (#99e1ea7) 
+:material-star:** Customer Request: GET Device by hostname**
 
-v1.4.0 - 2022-02-07
+We have improved our GET Devices REST API call with a new hostname query parameter.  By providing the hostname of the device with the $hostname= query parameter, MPS will return the device with the matching hostname.  This makes it easy to find the specific device connected to MPS if you know the hostname.
```

Comment:
is it really the $Hostname= ? if so, we need to change that pronto. $ is reserved for OData specification, $ should not be in the query parameter

Reply:
Our GET Devices REST API call now supports a new hostname query parameter. If you know the hostname, it is easy to find the specific device connected to MPS. Provide the hostname of the device with the `hostname=` query parameter, and MPS will return the device with the matching hostname.

Reply:
@rsdmike This is what is in Swagger and is used for all of the query parameters.  If that isn't correct, does Swagger need to be updated?

@Kimberly-Davis  updated

Reply:
@matt-primrose yeah, its not $ in the code, not sure why its that way in swagger